include/envoy/network/io_handle.h

```diff
@@ -1,33 +1,136 @@
 #pragma once
 
+#include <bitset>
 #include <memory>
 
 #include "envoy/common/pure.h"
 
 namespace Envoy {
+
+namespace Buffer {
```

Comment:
nit: prefer include header vs. forward decl unless there is a circular dependency, same below.